app/views/restaurants/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<p>This is the Show page for <%=@restaurant.name%> with test data to see if it properly connects to categories </p>
```

Comment:
Pretend this code is ready for clients. You shouldnt be saying that there is test data in this app